bin/install-core

```diff
@@ -120,13 +124,30 @@ mkdir -p "$DOWNLOAD_DIR"
 
 # TODO: uncomment the downloads when done testing other things
 # Download Bitcoin core checksums and signatures in background
-# TODO: these are messy make them legible.
-( cd "$DOWNLOAD_DIR"; #wget --tries=1 --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/download/ ||
-wget --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin https://bitcoincore.org/en/download/; ) & sig_dl=$!
+(
+    cd "$DOWNLOAD_DIR"
+    wget --tries=1 --no-host-directories --cut-dirs=1 --recursive \
+        #--level=1 --accept=SHA256SUMS* --include-directories=bin \
+        #"https://${BITCOIN_CORE_SOURCE_DOMAIN[0]}/en/download/" ||
+    wget --no-host-directories --cut-dirs=1 --recursive \
+        --level=1 --accept=SHA256SUMS* --include-directories=bin \
+        "https://${BITCOIN_CORE_SOURCE_DOMAIN[1]}/en/download/"
```

Comment:
Do you want to make a for loop. for 0..1; do wget ... ; done?

Or `for domain in ${BITCOIN_CORE_SOURCE_DOMAIN[*]}; do wget $domain/en/download/ ... ; done

Either is good.

Reply:
after wget needs && break, otherwise it'll try to download twice even if the first succeeds.

Reply:
Good idea. Fixed in f59ecac569ca591c342a954f81823cea1f080a63.